e2e/README.md

```diff
@@ -1,21 +1,8 @@
 These e2e tests depend on emulating or attaching a device.
 
-1) setup android stack. optionally you can do this with android studio, just make sure to name your emulators the way i did (29_gaps_32 and 30_ga_64)
-however, these command-line tools are easier to work with, once they're set up
-i followed these instructions
-https://web.archive.org/web/20230301233955/https://guides.codepath.com/android/installing-android-sdk-tools
-https://web.archive.org/web/20230301234131/https://stackoverflow.com/questions/65262340/cmdline-tools-could-not-determine-sdk-root/71765298
-
-to maintain shared resources with android studio (which had a java-error throwing version of sdkmanager)
-i installed the tools to `~/Android/Sdk/cmdline-tools/latest`
-so my `.bashrc` includes this:
-`export ANDROID_SDK_ROOT=$HOME/Android/Sdk
-export PATH=$PATH:$ANDROID_SDK_ROOT/emulator
-export PATH=$PATH:$ANDROID_SDK_ROOT/platform-tools
-export PATH=$PATH:$ANDROID_SDK_ROOT/cmdline-tools/latest/bin`
-
-2) install emulators
-`$ sh scripts/install_emulators.sh`
+1) setup android stack
+you can do this with android studio. create a api-level 30 x86_64 device and name it 30_ga_64
+alternatively, if you dont have android studio, this can help: https://github.com/fluidvanadium/androidstacker
```

Comment:
could you replace this line with the `install_emulator_for_e2e.sh` script for completeness?
will have to rebase anyway but if this is the only change we can merge if e2e fails anyway as i have see it pass. good job!

Reply:
done